azurelinuxagent/ga/exthandlers.py

```diff
@@ -87,7 +87,19 @@
 # This is the default sequence number we use when there are no settings available for Handlers
 _DEFAULT_SEQ_NO = "0"
 
+
+class HandlerStatus(object):
+    """
+    Statuses for Handlers
+    """
+    ready = "Ready"
+    not_ready = "NotReady"
+
+
 class ValidHandlerStatus(object):
```

Comment:
NIT: `HandlerStatus` and `ValidHandlerStatus` sounds very confusing. Maybe change the name of this class to `ExtensionStatus` or `ExtensionRuntimeStatus` or something to make it more clearer?

Reply:
Yes, but I plan to do that on a separate PR on develop. For now, i'm keeping it minimalistic for the release branch.